src/main/resources/initEnv

```diff
@@ -46,3 +46,11 @@ fi
 set -a
 eval "$VARS"
 set +a
+
+not_sourced() {
+	test "X$(basename -- "$0")" = "XinitEnv"
```

Comment:
**question**: What is the capital X for?

Reply:
A proof of how ugly shell scripts can be when needed to be written for the greatest compatibility ever. :)
https://stackoverflow.com/questions/19783948/portable-way-to-check-emptyness-of-a-shell-variable

Reply:
Context: This function was introduced in an old version of initEnv, but it was removed in my previous PR.
With the new feature added in this PR, the function was needed again, so I put it back.

I think in most cases the X padding is not needed, and I was not able to introduce any issues by removing it. However, I found some old forum threads recommending this format. Because initEnv is expected to run on many environments, my guess is that it was introduced to overcome a quirk of `test` in certain cases.

Reply:
Thanks for the explanation.